logging/logging.go

```diff
@@ -5,15 +5,24 @@ import (
 	"io"
 	"io/ioutil"
 	"log"
+	"os"
 	"strings"
 
-	"github.com/hashicorp/go-syslog"
+	gsyslog "github.com/hashicorp/go-syslog"
 	"github.com/hashicorp/logutils"
 )
 
 // Levels are the log levels we respond too.
 var Levels = []logutils.LogLevel{"TRACE", "DEBUG", "INFO", "WARN", "ERR"}
 
+// LogLevel is the log level of all logs in this package
+// it is set by the Setup() function and is shared with all other setup variations (i.e. SetupLocal())
+var LogLevel = ""
```

Comment:
Appreciate the context provided as a comment!

Reply:
Thanks!